Patch from markusg26@gmx.de to fix building of win32-based codecs (dmo and
quicktime).
See Gentoo Bug #132780.

Index: vlc-0.8.5/modules/codec/dmo/Makefile.am
===================================================================
--- vlc-0.8.5.orig/modules/codec/dmo/Makefile.am
+++ vlc-0.8.5/modules/codec/dmo/Makefile.am
@@ -99,27 +99,27 @@ endif
 
 libdmo_plugin_a_SOURCES = $(SOURCES_dmo)
 libdmo_plugin_la_SOURCES = $(SOURCES_dmo)
-libdmo_plugin_a_CFLAGS = `$(VLC_CONFIG) --cflags plugin dmo`
-libdmo_plugin_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags plugin dmo`
+libdmo_plugin_a_CFLAGS = `$(VLC_CONFIG) --cflags plugin dmo` -I../../../loader
+libdmo_plugin_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags plugin dmo` -I../../../loader
 libdmo_plugin_a_OBJCFLAGS = `$(VLC_CONFIG) --objcflags plugin dmo`
-libdmo_plugin_la_CFLAGS = `$(VLC_CONFIG) --cflags plugin dmo`
-libdmo_plugin_la_CXXFLAGS = `$(VLC_CONFIG) --cxxflags plugin dmo`
+libdmo_plugin_la_CFLAGS = `$(VLC_CONFIG) --cflags plugin dmo` -I../../../loader
+libdmo_plugin_la_CXXFLAGS = `$(VLC_CONFIG) --cxxflags plugin dmo` -I../../../loader
 libdmo_plugin_la_OBJCFLAGS = `$(VLC_CONFIG) --objcflags plugin dmo`
 libdmo_plugin_la_LDFLAGS = `$(VLC_CONFIG) --libs plugin dmo` \
 	-rpath '$(libvlcdir)' -module -shrext $(LIBEXT)
 libdmo_plugin_la_LIBADD = $(LTLIBVLC)
 
 libdmo_pic_a_SOURCES = $(SOURCES_dmo)
-libdmo_pic_a_CFLAGS = `$(VLC_CONFIG) --cflags builtin pic dmo`
-libdmo_pic_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin pic dmo`
+libdmo_pic_a_CFLAGS = `$(VLC_CONFIG) --cflags builtin pic dmo` -I../../../loader
+libdmo_pic_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin pic dmo` -I../../../loader
 libdmo_pic_a_OBJCFLAGS = `$(VLC_CONFIG) --objcflags builtin pic dmo`
 
 libdmo_a_SOURCES = $(SOURCES_dmo)
 libdmo_builtin_la_SOURCES = $(SOURCES_dmo)
-libdmo_a_CFLAGS = `$(VLC_CONFIG) --cflags builtin dmo`
-libdmo_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin dmo`
+libdmo_a_CFLAGS = `$(VLC_CONFIG) --cflags builtin dmo` -I../../../loader
+libdmo_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin dmo` -I../../../loader
 libdmo_a_OBJCFLAGS = `$(VLC_CONFIG) --objcflags builtin dmo`
-libdmo_builtin_la_CFLAGS = `$(VLC_CONFIG) --cflags builtin dmo`
-libdmo_builtin_la_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin dmo`
+libdmo_builtin_la_CFLAGS = `$(VLC_CONFIG) --cflags builtin dmo` -I../../../loader
+libdmo_builtin_la_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin dmo` -I../../../loader
 libdmo_builtin_la_OBJCFLAGS = `$(VLC_CONFIG) --objcflags builtin dmo`
 
Index: vlc-0.8.5/modules/codec/Makefile.am
===================================================================
--- vlc-0.8.5.orig/modules/codec/Makefile.am
+++ vlc-0.8.5/modules/codec/Makefile.am
@@ -519,28 +519,28 @@ librawvideo_builtin_la_OBJCFLAGS = `$(VL
 
 libquicktime_plugin_a_SOURCES = $(SOURCES_quicktime)
 libquicktime_plugin_la_SOURCES = $(SOURCES_quicktime)
-libquicktime_plugin_a_CFLAGS = `$(VLC_CONFIG) --cflags plugin quicktime`
-libquicktime_plugin_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags plugin quicktime`
+libquicktime_plugin_a_CFLAGS = `$(VLC_CONFIG) --cflags plugin quicktime` -I../../loader
+libquicktime_plugin_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags plugin quicktime` -I../../loader
 libquicktime_plugin_a_OBJCFLAGS = `$(VLC_CONFIG) --objcflags plugin quicktime`
-libquicktime_plugin_la_CFLAGS = `$(VLC_CONFIG) --cflags plugin quicktime`
-libquicktime_plugin_la_CXXFLAGS = `$(VLC_CONFIG) --cxxflags plugin quicktime`
+libquicktime_plugin_la_CFLAGS = `$(VLC_CONFIG) --cflags plugin quicktime` -I../../loader
+libquicktime_plugin_la_CXXFLAGS = `$(VLC_CONFIG) --cxxflags plugin quicktime` -I../../loader
 libquicktime_plugin_la_OBJCFLAGS = `$(VLC_CONFIG) --objcflags plugin quicktime`
 libquicktime_plugin_la_LDFLAGS = `$(VLC_CONFIG) --libs plugin quicktime` \
 	-rpath '$(libvlcdir)' -module -shrext $(LIBEXT)
 libquicktime_plugin_la_LIBADD = $(LTLIBVLC)
 
 libquicktime_pic_a_SOURCES = $(SOURCES_quicktime)
-libquicktime_pic_a_CFLAGS = `$(VLC_CONFIG) --cflags builtin pic quicktime`
-libquicktime_pic_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin pic quicktime`
+libquicktime_pic_a_CFLAGS = `$(VLC_CONFIG) --cflags builtin pic quicktime` -I../../loader
+libquicktime_pic_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin pic quicktime` -I../../loader
 libquicktime_pic_a_OBJCFLAGS = `$(VLC_CONFIG) --objcflags builtin pic quicktime`
 
 libquicktime_a_SOURCES = $(SOURCES_quicktime)
 libquicktime_builtin_la_SOURCES = $(SOURCES_quicktime)
-libquicktime_a_CFLAGS = `$(VLC_CONFIG) --cflags builtin quicktime`
-libquicktime_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin quicktime`
+libquicktime_a_CFLAGS = `$(VLC_CONFIG) --cflags builtin quicktime` -I../../loader
+libquicktime_a_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin quicktime` -I../../loader
 libquicktime_a_OBJCFLAGS = `$(VLC_CONFIG) --objcflags builtin quicktime`
-libquicktime_builtin_la_CFLAGS = `$(VLC_CONFIG) --cflags builtin quicktime`
-libquicktime_builtin_la_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin quicktime`
+libquicktime_builtin_la_CFLAGS = `$(VLC_CONFIG) --cflags builtin quicktime` -I../../loader
+libquicktime_builtin_la_CXXFLAGS = `$(VLC_CONFIG) --cxxflags builtin quicktime` -I../../loader
 libquicktime_builtin_la_OBJCFLAGS = `$(VLC_CONFIG) --objcflags builtin quicktime`
 
 # The subsdec plugin
